smithy-model/src/main/java/software/amazon/smithy/model/node/Node.java

```diff
@@ -155,6 +155,27 @@ public static String prettyPrintJson(Node node, String indentString) {
         return NodeHandler.prettyPrint(node, indentString);
     }
 
+    /**
+     * Writes the contents of a Node to a pretty-printed JSON string,
+     * and surrounds the result in backticks.
+     *
+     * @param node Node to write.
+     * @return Returns a string suitable for Markdown rendering.
+     */
+    public static String prettyPrintJsonWithBackticks(Node node) {
```

Comment:
I don't think this should be something on Node itself. I think this could go in ValidationUtils.